package.json

```diff
@@ -1,44 +1,52 @@
 {
-  "name": "tota11y",
-  "version": "0.1.6",
+  "name": "@khanacademy/tota11y",
+  "version": "0.2.0",
   "description": "An accessibility visualization toolkit",
-  "main": "build/tota11y.min.js",
+  "main": "dist/tota11y.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/Khan/tota11y.git"
   },
   "author": "Jordan Scales <scalesjordan@gmail.com>",
+  "contributors": [
+    "Jeff Yates <jeff.yates@alumni.manchester.ac.uk"
+  ],
+  "homepage": "https://khan.github.io/tota11y/",
   "devDependencies": {
     "accessibility-developer-tools": "2.11.0",
     "autoprefixer": "^6.5.3",
     "babel": "^6.23.0",
     "babel-core": "^6.26.3",
-    "babel-loader": "^6.4.1",
+    "babel-loader": "^7.1.5",
     "babel-preset-env": "^1.7.0",
     "babel-preset-react": "^6.24.1",
-    "css-loader": "^0.10.1",
+    "clean-webpack-plugin": "^2.0.1",
+    "css-loader": "^2.1.1",
     "eslint": "^5.15.2",
-    "handlebars": "^4.1.0",
-    "handlebars-loader": "^1.0.0",
+    "handlebars": "^4.1.2",
+    "handlebars-loader": "^1.7.1",
     "jquery": "^3.3.1",
     "jsdom": "^8.1.0",
-    "less": "^2.5.0",
-    "less-loader": "^2.2.0",
-    "mocha": "^6.0.2",
+    "less": "^3.9.0",
+    "less-loader": "^4.1.0",
+    "mocha": "^6.1.4",
     "postcss": "^5.2.6",
-    "postcss-loader": "^1.1.1",
-    "script-loader": "^0.6.1",
-    "style-loader": "^0.10.1",
-    "webpack": "^1.8.4",
-    "webpack-dev-server": "^1.9.0"
+    "postcss-loader": "^3.0.0",
+    "script-loader": "^0.7.2",
+    "style-loader": "^0.23.1",
+    "uglifyjs-webpack-plugin": "^2.1.2",
+    "webpack": "^4.30.0",
+    "webpack-cli": "^3.3.0",
+    "webpack-dev-server": "^3.3.1"
   },
   "scripts": {
-    "build": "npm run prod && npm run dev",
-    "prod": "NODE_ENV=production webpack --config webpack.config.babel.js",
-    "dev": "webpack --config webpack.config.babel.js -d --devtool hidden --output-filename=tota11y.js",
+    "build:prod": "NODE_ENV=production webpack --config webpack.config.babel.js",
+    "build:dev": "webpack --config webpack.config.babel.js -d --devtool hidden",
     "lint": "eslint index.js plugins test utils",
     "test": "mocha --require test/babel-hook test/*.js",
-    "live-test": "webpack-dev-server --config webpack.config.babel.js --hot --inline"
+    "test:live": "webpack-dev-server --config webpack.config.babel.js --hot --inline",
+    "start": "npm run test:live",
```

Comment:
What's the use of `test:live` if it's the same thing as `start`?

Reply:
Great point!